theme.json

```diff
@@ -156,11 +180,21 @@
 					"size": "clamp(3rem, 5vw + 1.5rem, 6.029rem)",
 					"slug": "3x-large"
 				},
+				{
+					"name": "3.5X Large",
+					"size": "clamp(64px, calc(4rem + ((1vw - 4.8px) * 6.3889)), 110px);",
```

Comment:
you might want to consider using the [built-in tools](https://make.wordpress.org/core/2022/05/27/block-font-sizes-and-fluid-typography/) for fluid typography